### ISSUE TITLE
Skills are 5e skills

### ISSUE THREAD

**Issue by matty-dee** [encounterplus/pf2e#1]
When creating a creature, Crafting is missing as a skill.

All skills also display as "Skill.[Skill name]"

**Comment by matty-dee** [comment]
Just realized, this is due to all skills being 5e skills -- not Pf2e

**Comment by jurex** [comment]
yeah, it's just copy paste used during rapid prototyping :D thanks for corrections guys!
